docker-compose-cli.yaml

```diff
@@ -0,0 +1,75 @@
+version: '3.2'
+
+services:
+  app:
+    build:
+      context: .
+      dockerfile: ./packages/app/Dockerfile
+    command: npm run --prefix packages/app dev -- --host
```

Comment:
why dev? For cli I'd suggest build + start

Reply:
Fixed